internal/ingress/controller/template/template.go

```diff
@@ -1186,15 +1186,16 @@ func buildAuthSignURLLocation(location, authSignURL string) string {
 }
 
 var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+var random *rand.Rand
```

Comment:
Fixing the deprecated rand. I'm also wonder if we can replace `math/rand` with `crypto/rand`

Reply:
Replacing with crypto/rand when required sounds better IMO